source/parser/intermediate-instruction.cpp

```diff
@@ -188,6 +189,31 @@ void IntermediateInstruction::insertIntoArguments(const std::string& name,
   replaceInVector(_targets, name, value);
 }
 
+std::string IntermediateInstruction::toString() const {
+    std::string str = _name;
+
+    // Append targets
+    for (size_t i = 0; i < _targets.size(); i++) {
+      if (i != 0)
```

Comment:
I guess you didn't write this, but either way there should be braces around these `if`/`else`s